<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IpFreelyDownloadWidget</class>
 <widget class="QFrame" name="IpFreelyDownloadWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>109</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="styleSheet">
   <string notr="true">#IpFreelyDownloadWidget {
  background: palette(button);
  border: 1px solid palette(dark);
  margin: 0px;
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <layout class="QGridLayout" name="m_topLevelLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="dstName">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="srcUrl">
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
